pkg/README

```diff
@@ -1,20 +0,0 @@
-# New Go Implementation of the Remote ID portion of the DSS.
```

Comment:
Let's update this README rather than delete -- we don't want to regress on amount of documentation.

Reply:
This readme is super old. All of the content here has been captured in the newer readme in the root of the directory

Reply:
root of the repo*